.github/workflows/push-tag.yaml

```diff
@@ -0,0 +1,44 @@
+name: Push Tag
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - "main"
+
+permissions:
+  contents: write
+
+jobs:
+  push-tag:
+    if: github.event.pull_request.merged == true && startsWith(github.head_ref, 'release/')
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Generate a token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.APP_ID_OF_YUMEMI_PR_TOKEN_GENERATOR }}
+          private-key: ${{ secrets.APP_PRIVATE_KEY_OF_YUMEMI_PR_TOKEN_GENERATOR }}
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ steps.generate_token.outputs.token }}
+      - name: Extract a version
+        id: extract_version
+        run: |
+          version=$(echo ${{ github.head_ref }} | sed -e 's/release\///')
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey)
`version=`: variable definition
`$(xxx)`: use the value of the execution result
`sed -e 's/release\///'`: replace `release` with a space
https://qiita.com/shunbaba/items/faf0cf2dbfd4e205e945